tendrl/commons/manager/__init__.py

```diff
@@ -14,27 +14,35 @@ def __init__(
             self,
             sds_sync_thread,
             central_store_thread,
+            message_handler_thread=None,
     ):
         self._central_store_thread = central_store_thread
         self._sds_sync_thread = sds_sync_thread
         self._job_consumer_thread = jobs.JobConsumerThread()
+        self._message_handler_thread = message_handler_thread
 
     def stop(self):
         LOG.info("%s stopping" % self.__class__.__name__)
```

Comment:
@brainfunked This log message is removed in @anmolsachan patch because it is happening before node socket start